ipatests/pytest_plugins/integration/create_external_ca.py

```diff
@@ -25,12 +25,17 @@
 import datetime
 import six
 
+ISSUER_CN = 'example.test'
 
 class ExternalCA(object):
     """
     Provide external CA for testing
     """
-    def create_ca(self, cn='example.test'):
+    def __init__(self, days=365):
+        self.now = datetime.datetime.utcnow()
+        self.delta = datetime.timedelta(days=days)
```

Comment:
``now`` and ``delta`` are not used in this branch. Please remove ``def __init__`` again.

Reply:
removed.
Thanks.